### PR TITLE
Completed prep-javascript-arrays - Victoria Dixon

### PR DESCRIPTION
**Pull request by Victoriakdixon88**
Screen Shot 2024-01-11 at 8 55 09 PM

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+const firstElement = languages [0];
+console.log(firstElement);
+
+const thirdElement = languages [2];
+console.log(thirdElement);
+
+const length = languages.length;
+console.log(length);
+
+const lastElement =languages [5];
```
Comment (BrettAlbright) [inline prep-javascript-arrays/main.js]: this element should be accessed by calculating the end of the array using the length variable created on line 10. Please reach out if you have any questions!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
One updated needed!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Comment by Victoriakdixon88** [comment]
updated and revised 
Screen Shot 2024-01-12 at 1 46 32 PM
Screen Shot 2024-01-12 at 1 37 34 PM

**Pull request merged**